tests/dashboard/dashboard-onboarding-admin.spec.js

```diff
@@ -80,23 +82,25 @@ test.describe('Onboarding Admin page tests', () => {
     await expect(table.locator('tbody')).toBeVisible()
     const rows = await table.locator('tbody').locator('tr').all()
 
-    const firstUser = rows.slice(0, 2)
+    const steps = Object.keys(PORTAL_USER_ACCOUNT_SETUP_STEPS).length;
```

Comment:
Should be changed to this
```suggestion
    const steps = portalUserAccountSteps.length;
```